README.md

```diff
@@ -83,6 +83,10 @@ Defaults matter _a lot_, but clear and well-executed processes that allow employ
 
 [ContractPatch](https://sfconservancy.org/contractpatch/), information about negotiating employment agreements for open source developers.
 
+#### Shared ownership
+
+[SourceHut documents](https://sourcehut.org/blog/2022-10-09-ip-assignment-or-lack-thereof/) an approach wherein IP is held collectively by all of its employees and the free software license the software is distributed under is used to provide the appropriate rights to all parties. This approach ensures the employees right to co-ownership over their work and increases the staying power of the free software licenses used. This ensures that the workers and the employer have equal ownership and equal rights to the commercialization of the IP.
+
```

Comment:
How does this effect licensing changes? For example, moving from GPLv2 to GPLv3 or v3.0 to v3.1?

Reply:
You cannot change licenses without the agreement of all participants under this model (feature, not a bug)

Reply:
@ddevault it cuts both ways. Linux kernel right now has some code that they cannot change the license on because the developer passed away and the estate doesn't understand things. So they have to rewrite the code if they want to change the license. Other projects like FSF/GNU and Qt require Copyright License Agreements (CLAs) that assign the copyright over to an organization so they can manage the licenses. Most live in the murkiness of licensing like the Linux Kernel does. So just realize there are Pros and Cons to both approaches and no a single one-size-fits-all methodology.

FYI - Qt does it because they are dual licensed with a commercial license. FSF/GNU does it because they upgrade all the use of licenses on all projects they support/operate.

So I'm not sure this is quite a good place to have that. Perhaps creating a section that could offer optional amendments that some may want to adopt would be a good idea for something like this.

Reply:
This makes the company code work like any project that lacks a CLA: You can't change license unless the original license permits you to.

(Eg going from MIT to GPL would generally be considered possible, and any code licenses with eg. "GPL 2 or later")

This can be important if the employment contract was signed with the intention of working for a copyleft project and avoiding the company later reversing on that decision and stopping sharing their code.

I think it belongs here.